### PR TITLE
Adding homepage and mailing list to the README

### PR DESCRIPTION
**Pull request by nmittler**

### CODE REVIEW

**Comment by nmittler** [comment]
@ejona86 PTAL

/cc @wrwg

File: README.md [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
-[![Coverage Status](https://coveralls.io/repos/grpc/grpc-java/badge.svg?branch=master&service=github)](https://coveralls.io/github/grpc/grpc-java?branch=master)
-
 gRPC-Java works with JDK 6. TLS usage typically requires using Java 8, or Play
 Services Dynamic Security Provider on Android. Please see the [Security
 Readme](SECURITY.md).
 
+<table>
+  <tr>
+    <td><b>Homepage:</b></td>
+    <td><a href="http://www.grpc.io/">www.grpc.io</a></td>
```
Comment (carl-mastrangelo) [inline README.md]: https?

Comment (nmittler) [inline README.md]: doesn't seem to work

Comment (ejona86) [inline README.md]: Yeah, https doesn't work. It is certificate issues.

**Comment by ejona86** [comment]
@nmittler LGTM

**Pull request merged**